app/styles/spacing.scss

```diff
@@ -72,4 +72,16 @@ $space-map: (
   .width-#{$size} {
     width: $space;
   }
+
+  // Space
+  .space-y-#{$size} {
+    & > :not([hidden]) ~ :not([hidden]) {
+      margin-top: $space;
+    }
+  }
+  .space-x-#{$size} {
+    & > :not([hidden]) ~ :not([hidden]) {
+      margin-left: $space;
```

Comment:
is this supposed to be margin left? doesn't seem to make sense with "x"? I thought x would be both left and right?